### PR TITLE
Skip audit on npm install

### PR DESCRIPTION
**Pull request by st3iny**
Reduce time to install each app. This will skip checking for vulnerabilities and printing a summary.

### CODE REVIEW

File: remote.sh [hunk 1]
```diff
@@ -243,12 +243,12 @@ if [ -n "$BRANCH" ] && ! [ -f "/var/www/$APPID-completed" ]; then
 
     # Fix npm ci for mail
     if [ "$APPID" = mail ]; then
-        npm install
+        npm install --no-audit
```
Comment (szaimen) [inline remote.sh:246]: btw, @st3iny would it be possible to remove the workaround for mail by commiting a usable package-lock.json file?

**Review by szaimen — APPROVED** [review]
LGTM, thanks! :)

File: remote.sh (hunk 1 above)
Comment (st3iny) [reply remote.sh:246]: It should be usable. I just tried it locally and it works fine on node 16 and npm 7.

Comment (szaimen) [reply remote.sh:246]: All right! Thanks @st3iny :)

Just removed it with https://github.com/szaimen/nextcloud-easy-test/commit/bc653f5e5497d1ddc9a6a8c8e269260107504caa

**Pull request merged**